indigo_app/templates/indigo_api/workflow_list.html

```diff
@@ -73,7 +73,7 @@
                     <div class="progress-bar bg-success" role="progressbar" style="width: {{ workflow.pct_complete }}%"></div>
                   </div>
                 </div>
-                <div class="col-3 text-muted">{{ workflow.task_counts.complete }} of {{ workflow.task_counts.total }} task{{ workflow.task_counts.total|pluralize }}</div>
+                <div class="col-3 text-muted">{{ workflow.task_counts.done|default:'0'}} of {{ workflow.task_counts.total }} task{{ workflow.task_counts.total|pluralize }}</div>
```

Comment:
I'm skeptical about this. The progress bar considers "cancelled" tasks as complete, but this doesn't. So the progress bar could be at 100%, but this could be at "8 of 10" if tasks have been cancelled.

Reply:
my intention was for the progress bar to only count 'done' tasks as complete – 
`w.pct_complete = w.task_counts.get('done', 0) / (w.task_counts['total'] or 1) * 100.0`
and
`w.task_counts['total'] = sum(x for x in w.task_counts.itervalues()) - \
                                     w.task_counts.get('cancelled', 0)`